.containers/K1/docker/Dockerfile

```diff
@@ -36,8 +36,8 @@ RUN apt -y install curl
 RUN curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py
 RUN python2 get-pip.py
 
-RUN pip2 install python-igraph
-RUN pip2 install scikit-learn
+RUN pip2 install python-igraph==0.8.3
```

Comment:
Do you mind adding these version specifications to the Singularity file too (at `.containers/K1/singularity/Singularity`)?